File: core-java/src/main/java/com/baeldung/datetime/UseZonedDateTime.java
package com.baeldung.datetime;

import java.time.LocalDateTime;
import java.time.ZoneId;
import java.time.ZonedDateTime;

class UseZonedDateTime {

    ZonedDateTime getZonedDateTime(LocalDateTime localDateTime, ZoneId zoneId) {
        return ZonedDateTime.of(localDateTime, zoneId);
    }
}

File: core-kotlin-modules/core-kotlin-date-time/src/main/kotlin/com/baeldung/dates/datetime/UseZonedDateTime.kt
package com.baeldung.dates.datetime

import java.time.LocalDateTime
import java.time.ZoneId
import java.time.ZonedDateTime

class UseZonedDateTime {

    fun getZonedDateTime(localDateTime: LocalDateTime, zoneId: ZoneId): ZonedDateTime {
        return ZonedDateTime.of(localDateTime, zoneId)
    }
}